samples/bluetooth/direction_finding_central/child_image/hci_rpmsg.conf

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_BT_CTLR=y
+CONFIG_BT_LL_SW_SPLIT=y
+
+# Enable new implementation of LLCPs
+CONFIG_BT_LL_SW_LLCP=y
+
+# Enable Direction Finding Feature including AoA and AoD
+CONFIG_BT_CTLR_DF=y
+
+CONFIG_BT_CTLR_DF_CTE_RX=y
+CONFIG_BT_CTLR_DF_CONN_CTE_RX=y
+CONFIG_BT_CTLR_DF_ANT_SWITCH_RX=y
+CONFIG_BT_CTLR_DF_CONN_CTE_REQ=y
+
+# Ensure that there is enough control prcedure contexts to queue and execute all procedures
+CONFIG_BT_CTLR_LLCP_PROC_CTX_BUF_NUM=6
+CONFIG_BT_CTLR_ADVANCED_FEATURES=y
+
+# Increase size of buffer for events received by Host. Size must be enought to store
+# comple IQ samples report.
```

Comment:
```suggestion
# the complete IQ samples report.
```

Reply:
These sentence points to one of many `complete IQ samples reports`, not particular one, hence it seems to me that `the` should not be here.

Reply:
It also possible to use the indefinite article "a" instead of "the" in my suggestion.